Octokit.Tests/Reactive/ObservablePullRequestsClientTests.cs

```diff
@@ -225,7 +225,7 @@ public class TheMergeMethod
             [Fact]
             public void MergesPullRequest()
             {
-                var mergePullRequest = new MergePullRequest { CommitMessage = "fake commit message" };
+                var mergePullRequest = new MergePullRequest { CommitMessage = "fake commit message", CommitTitle = "fake title" };
```

Comment:
No this is not necessary. but i thought i should also make changes in ObservablePullRequestsClientTests.cs .I wasn't sure that i should add property here or leave it as it was.. what do you say should i remove this?